tetrateci/create_aks_cluster.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+CLUSTER_NAME="test-istio-$GITHUB_SHA"
+az aks create --resource-group getistio --name $CLUSTER_NAME --node-count 2 --generate-ssh-keys # change the resource group
```

Comment:
Here and elsewhere cross check if the cluster_name resource already exists. If so, delete throw a message into the log, delete the previous cluster and then recreate it. This is to make sure we start from a clean state.

Reply:
This won't be possible, github sha would be different with every commit and also the post cleanup would clean up the cluster even if it fails